gen/parser-util.go

```diff
@@ -98,21 +99,30 @@ func vgKeyExpr(n *html.Node) string {
 // 	return ""
 // }
 
-func vgForExpr(n *html.Node) string {
-	for _, a := range n.Attr {
-		if a.Key == "vg-for" {
-
-			v := strings.TrimSpace(a.Val)
-
-			// moved to ParserGo.emitForExpr
-			// if !strings.Contains(v, " ") { // make it so `w` is a shorthand for `key, value := range w`
-			// 	v = "key, value := range " + v
-			// }
+type vgForAttr struct {
+	expr     string
+	noshadow bool
+}
 
-			return v
+func vgForExpr(n *html.Node) (vgForAttr, error) {
+	for _, a := range n.Attr {
+		if strings.HasPrefix(a.Key, "vg-for") {
```

Comment:
This will also match `vg-forabcdef`, better would be to change it to `a.Key == "vg-for" || strings.HasPrefix(a.Key, "vg-for.")`.  But I will go ahead and merge and just do this one change after.